Sources/StepperFilter/StepperFilterViewController.swift

```diff
@@ -35,7 +35,11 @@ public class StepperFilterViewController: UIViewController, FilterContainerViewC
     public override func viewDidLoad() {
         super.viewDidLoad()
         view.addSubview(stepperFilterView)
-        stepperFilterView.fillInSuperview()
+        NSLayoutConstraint.activate([
+            stepperFilterView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            stepperFilterView.topAnchor.constraint(equalTo: view.topAnchor, constant: 148),
```

Comment:
I don't like that `148` magic value... does it work on iPhone 4S? 😛 

Is there no other way to position it without magic constants? Or calculate it based on height?

Reply:
Haha, I'll figure something out! 😄

Reply:
`.veryLargeSpacing * 2`  😜

Reply:
Looks like that works well 😄